src/content/protocols/sky.md

```diff
@@ -0,0 +1,387 @@
+
+# Overview
+
+## Chain
+
+The Sky protocol is currently mainly on Ethereum mainnet. As part of Sky's _End Game_ plan the protocol is gradually deploying on other chains and aims to launch its own chain in the future. This review focuses on the Ethereum mainnet deployment of Sky.
+
+> Chain score: Low
+
+## Upgradeability
+
+Core contracts such as `USDS` and `sUSDS` are upgradeable through governance actions. Governance has a complete control over the protocol and could arbitrarily mint tokens, liquidate positions, stop contracts, and steal collateral.
+Governance is controlled by `MKR`/`SKY` token holders who sealed their tokens to earn voting power which allows them to submit and vote on respective proposals, or delegate their votes to another user.
+
+> Upgradeability score: High
+
+## Autonomy
+
+The Sky protocol relies on the provider Chronicle for price feeds of collateral assets. An Oracle Security Module (`OSM`) enforces a 1-hour delay on price updates and the governance can freeze the current price to prevent further updates in case of emergency. The Chronicol protocol uses non-upgradeable contracts and a system of decentralized validators (and challengers) to push price updates on-chain. The price feed provider could be changed through a governance proposal.
+
+> Autonomy score: Medium
+
+## Exit Window
+
+The minimum delay between approval and execution of a governance proposal is currently **18 hours**. Governance proposals have a recurring weekly and monthly schedules. Those proposals usually have a courtesy delay duration of 2-3 days before taking effect, but proposals remain possible at any time with the minimum delay. Proposals may (but do not have to) include an office-hours modifier that means it can only be executed between 14:00 and 21:00 UTC Monday-Friday. There are no minimum amount of tokens to open a proposal or for it to pass, as the protocol uses continuous approval. This is explained in more details in the [exit window analysis section](#exit-window-1).
+
+Emergency measures allow the governance to pause certain contracts through a governance proposal without being subject to the mandatory delay. In addition to that an _Emergency Shutdown Module_ (ESM) exists and can irreversibly shutdown the entire protocol if **500'000** `MKR` tokens are sent to the Emergency Shutdown Contract. Funds sent to the contract are considered lost no matter the emergency status.
+
+> Exit Window score: High
+
+## Accessibility
+
+Sky has a main frontend at [sky.money](https://sky.money). The frontend is not self-hostable nor open source, but multiple other access points exist with _stars_ such as [Spark](https://spark.fi) which is self-hostable or third-party apps like DeFiSaver or SummerFi. These apps build an acceptable backup solution in case of failure of the official frontend.
```

Comment:
lets not use unkown terms like "stars" here

is SummerFi compatible with positions created through sky.money? afaik summerFi uses "smart accounts" as the actual owners of a position making it impossible to eg close a position that was opened on the main UI through SummerFi